docs/_sass/custom/custom.scss

```diff
@@ -225,6 +225,14 @@ body {
 		background-color: #ffffff; color: #279890;
 	}
 
+	.note-small {
+		background-color: $whitegrey;
+		border-left: 4px solid $aqua-marine;
+		border-radius: 0px 2px 2px 0px;
+		padding: 4px 10px;
+		font-size: medium;
+	}
```

Comment:
now that we have [proper callouts](https://docs.lakefs.io/callouts.html), can we just use those instead of more custom CSS?